cmd/kubeadm/app/util/system/package_validator.go

```diff
@@ -92,30 +90,30 @@ func (validator *packageValidator) Name() string {
 // Validate checks packages and their versions against the spec using the
 // package manager on the running machine, and returns an error on any
 // package/version mismatch.
-func (validator *packageValidator) Validate(spec SysSpec) (error, error) {
+func (validator *packageValidator) Validate(spec SysSpec) ([]error, []error) {
 	if len(spec.PackageSpecs) == 0 {
 		return nil, nil
 	}
 
 	var err error
 	if validator.kernelRelease, err = getKernelRelease(); err != nil {
-		return nil, err
+		return nil, []error{err}
```

Comment:
this does not feel great too me. wondering if there is a better way of doing this.

Reply:
Looks like the best we can do ATM.